### PR TITLE
Services and Repository tests

### PR DESCRIPTION
**Pull request by ghost**
@kilisoria @mravinale

### CODE REVIEW

File: Source/Hiperion/Hiperion/Services/UserServices.cs [hunk 1]
```diff
@@ -7,30 +9,33 @@
...
 
+    #endregion
+
     public class UserServices : IUserServices
     {
+        private readonly IMappingEngine _mapperEngine;
         private readonly IUserRepository _repository;
 
-        public UserServices(IUserRepository repository)
+        public UserServices(IUserRepository repository, IMappingEngine mapperEngine)
```
Comment (mravinale) [inline Source/Hiperion/Hiperion/Services/UserServices.cs:19]: great!

File: Source/Hiperion/Hiperion/Infrastructure/Ioc/WindsorInstaller.cs [hunk 2]
```diff
@@ -10,13 +13,16 @@
...
+
     public class WindsorInstaller : IWindsorInstaller
     {
         public void Install(IWindsorContainer container, IConfigurationStore store)
         {
             var connectionString = ConfigurationManager.ConnectionStrings["HiperionDb"].ConnectionString;
-
             container.Register(
+                Component.For<IMappingEngine>()
+                    .Instance(Mapper.Engine),
```
Comment (mravinale) [inline Source/Hiperion/Hiperion/Infrastructure/Ioc/WindsorInstaller.cs:25]: awesome!

**Comment by mravinale** [comment]
great work with mappings!

**Comment by ghost** [comment]
:)

**Pull request merged**